HelpMyStreet.Utils/HelpMyStreet.Utils/Extensions/RequestTypeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using HelpMyStreet.Utils.Enums;
+using System;
+
+namespace HelpMyStreet.Utils.Extensions
+{
+    public static class RequestTypeExtensions
+    {
+        public static string PerfectTense(this RequestType requestType, int count)
+        {
+            if (count <= 1)
```

Comment:
Better to check for == 1 here, as for zero the correct response would be 0 shift**s** completed